### PR TITLE
Remove flux

### PR DESCRIPTION
**Pull request by mertdy**
Since the flux is deprecated, I removed it from the project and simulated a mini Dispatcher class. @Kikobeats

### CODE REVIEW

**Comment by Kikobeats** [comment]
Excellent <picture data-single-emoji=":excellent-mrburns:" title=":excellent-mrburns:">:excellent-mrburns:</picture>

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@microlink/react-json-view",
   "description": "Interactive react component for displaying javascript arrays and JSON objects.",
   "homepage": "https://github.com/microlinkhq/react-json-view",
-  "version": "1.23.4",
+  "version": "1.24.0",
```
Comment (Kikobeats) [inline package.json:5]:
```suggestion
  "version": "1.23.4",
```

**Comment by Kikobeats** [comment]
@mertdy any idea how to fix this?
https://github.com/microlinkhq/react-json-view/actions/runs/12495308704/job/34865895522

**Comment by mertdy** [comment]
@Kikobeats I am sure the following lines cause the error:

test/testHelpers/requireSources.js
(starting line 17)
image

'navigator' is already defined in 'global' and cannot be modified.

Maybe you should remove this line since global.navigator.userAgent already gives something like 'node.js' (not the exact string but similar). However, I cannot guarantee that the removal will resolve the problem.

**Pull request merged**